production/ksonnet/loki/distributor.libsonnet

```diff
@@ -15,14 +15,16 @@
     container.mixin.readinessProbe.httpGet.withPort($._config.http_listen_port) +
     container.mixin.readinessProbe.withInitialDelaySeconds(15) +
     container.mixin.readinessProbe.withTimeoutSeconds(1) +
-    $.util.resourcesRequests('500m', '500Mi') +
-    $.util.resourcesLimits('1', '1Gi'),
+    $.util.resourcesRequests($._config.distributor.CPURequests, $._config.distributor.memoryRequests) +
+    $.util.resourcesLimits($._config.distributor.CPULimits, $._config.distributor.memoryLimits),
 
   local deployment = $.apps.v1.deployment,
 
   distributor_deployment:
-    deployment.new('distributor', 3, [$.distributor_container]) +
+    deployment.new('distributor', $._config.distributor.replicas, [$.distributor_container]) +
+    $.extra_tolerations +
```

Comment:
personally i would prefer to use `deployment.spec.template.spec.withTolerations($._config.tolerations)` here directly since in this case we are not calling a _complex function_ but just wrapping a one liner in one line

Reply:
i tried to keep the same approach used in `config_hash_mixin`. i will change it and add labels as well.

Reply:
@primeroz do you have any other comment? I will resolve these conversation and ask to you for approval to merge it. Please let me know how can I help here.

Reply:
I can't approve, i am just a user of this code :)

Reply:
okay. i will close the reviews @primeroz 
thanks for helping me.